catalyst-service/src/main/resources/application.yml

```diff
@@ -1074,6 +1077,6 @@ initializr:
       id: java
       default: true
   bootVersions:
-    - name: 1.5.10
-      id: 1.5.10.RELEASE
+    - name: 2.0.1
+      id: 2.0.1.RELEASE
```

Comment:
Currently, the latest release is **v2.0.4.RELEASE** - let's follow Spring Boot's versions